extensions/ql-vscode/src/packaging.ts

```diff
@@ -82,26 +83,44 @@ interface QLPackQuickPickItem extends QuickPickItem {
 export async function handleInstallPackDependencies(
   cliServer: CodeQLCliServer,
   progress: ProgressCallback,
+  relativeDirectoryPath?: string,
 ): Promise<void> {
   progress({
     message: "Choose packs to install dependencies for",
     step: 1,
     maxStep: 2,
   });
-  const workspacePacks = await cliServer.resolveQlpacks(
-    getOnDiskWorkspaceFolders(),
-  );
-  const quickPickItems = Object.entries(
-    workspacePacks,
-  ).map<QLPackQuickPickItem>(([key, value]) => ({
-    label: key,
-    packRootDir: value,
-  }));
-  const packsToInstall = await window.showQuickPick(quickPickItems, {
-    placeHolder: "Select packs to install dependencies for",
-    canPickMany: true,
-    ignoreFocusOut: true,
-  });
+
+  let packsToInstall: QLPackQuickPickItem[] | undefined;
+
+  if (relativeDirectoryPath) {
+    const dirPath = resolve(
+      join(workspace.rootPath || "", relativeDirectoryPath),
```

Comment:
There are some edge cases that I am not sure are covered here. 

If this is a multi-root workspace, what is the value of `workspace.rootPath`? What value should we even use? If `workspace.rootPath` is empty or undefined, does that necessarily mean that `relativeDirectoryPath` is absolute?